pkg/ext/stores/passwordchangerequest/store.go

```diff
@@ -0,0 +1,207 @@
+package passwordchangerequest
+
+// tokens implements the store for the new public API token resources, also
+// known as ext tokens.
```

Comment:
password change requests, not tokens

Reply:
fixed [`12a3bb3` (#50740)](https://github.com/rancher/rancher/pull/50740/commits/12a3bb321589b1ab9c83851e8f17c5f5f84f673d)